### ISSUE TITLE
PR curve

### ISSUE THREAD

**Issue by My1new** [kalenforn/DCHMT#6]
Hello, I have a question, which method of your code you use to draw the PR curve, is it 'calc _ precisions _ hash _ my ', and what is your Gnd parameter

**Comment by kalenforn** [comment]
Gnd is the similarity matrix in the training step. Gnd = label x label. Notice, I am not ensuring this function will work, please refer to [here](https://github.com/WangGodder/deep-cross-modal-hashing/blob/master/torchcmh/training/valid.py).
